app/views/admin/get_involved/index.html.erb

```diff
@@ -1,9 +1,26 @@
-<% page_title 'Get involved' %>
+<% content_for :page_title, "Get involved" %>
+<% content_for :title, "Get involved" %>
+<% content_for :title_margin_bottom, 2 %>
 
-<div class="get-involved-header">
-  <h1>Get involved</h1>
-  <%= link_to "View on website", get_involved_url %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">
+      <%= link_to "View on website", get_involved_url, class: "govuk-link" %>
```

Comment:
Please adjust View on Website link to match latest in the Guide to match Carlos's approved style. 
Latest commit here: https://github.com/alphagov/whitehall/pull/7487

Reply:
So:

```suggestion
      <%= link_to "View on website", get_involved_url, class: "govuk-link", target: "_blank" %>
```

Is that correct?

Reply:
I've updated the spacing and the link target:

![Screenshot 2023-03-28 at 2 58 50 pm](https://user-images.githubusercontent.com/4599889/228286972-0cc09b40-d956-4bce-b5cc-8acb019c4342.png)